.github/workflows/android-build-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test android build
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'yarn'
+      - name: Install dependencies
```

Comment:
In iOS and TvOS builds `Install node dependencies` name was used. Both of them are fine but stick to one